### ISSUE TITLE
Multi-account proxying

### ISSUE THREAD

**Issue by theotheroracle** [ajbura/cinny#1071]
### Describe the problem

i have to use seperate clients for each of my headmates for example, firefox multi account containers, or epiphany sandboxes . 

### Describe the solution you'd like

an ideal solution, would be much like the discord bot pluralkit, or for a matrix implentation, moment, or more obscurely, fluffychat bundles . 

how this would work ideally, is there is a option to sign in with multiple accounts, ( depends on #66 ), and then have an easy way to switch between them, for example with an icon next to the enter field, but ideally with user-defined prefixes, such as "m. " or "mary [ " for example . when these would be typed, the client would switch to that account if it's already in the room, and if not, ideally ask to join the room, if the room is public, or if a user in the room has permission to invite ( you'd probably want to make this toggleable though, i could see some not appreciating this )

this would make the experience much nicer for plural systems <3

### Alternatives considered

as mentioned above, alternatively you can use mutiple clients which doesn't work on computers with limited ram, or you could do as hydrogen does and allow accounts to be switched within the client, however this is still quite cumbersome and difficult to do quickly which can disrupt the flow of conversation and cause burnout over time .

### Additional context

for context, plural systems often want to be able to quickly switch between multple voices to represent different headmates voices and thoughts in the natural flow of conversation . for some systems, only one headmate is active at a time, and such quick switching is irrelevant to them, but for many this is not the case, and features like this would be quite useful . perhaps for those without such needs, simple hydrogen style multi-account switching would be sufficient .

**Comment by viasux** [comment]
also good for roleplayers / people who just manage multiple accounts!

**Comment by LinuxinaBit** [comment]
An example of a good multi-account switcher, funnily enough, is Discord’s implementation:

![B874DF01-4668-48E8-988E-DDEC348DE1EF](https://user-images.githubusercontent.com/105567407/221462244-613629f2-8cdc-4cba-8c5f-5de9fb650272.png)

In Cinny, the profile picture in the bottom left currently acts as a settings menu button.
A pop up could appear instead, with a list of accounts, an option to add a new one, and a button for the settings menu.

A less drastic change could just be to add a list at the top of the settings in the blank space here:

![B544D180-563A-46E7-BD13-4464E63CE22D](https://user-images.githubusercontent.com/105567407/221462644-906db336-3089-4ccd-9538-310d4ee14a9a.jpeg)

**Comment by csingewald** [comment]
Cinny is great! I think this feature is important because many people have different professional and personal accounts.

**Comment by parthamjangir2020** [comment]
Right multi account complete features is awesome

**Comment by artemislena** [comment]
DeM: We appreciate the mention of FluffyChat's bundles and would like to see it used for Cinny's multiaccount feature, if one gets implementd, as well, as that would allow individuals and plural groups to have consistent setups across several clients, rather than having to set up the same thing for each client individually.

**Comment by kfiven** [comment]
@artemislena Have the FluffyChat folks created an MSC for the feature?

**Comment by artemislena** [comment]
DeM: I am afraid not. The original developer of the feature stopped working on FluffyChat after implementing it (in fact, she does not even use it anymore), and [other developers are not using or maintaining the feature](https://gitlab.com/famedly/fluffychat/-/issues/1186), leading to [external contributors having to fix issues with it](https://gitlab.com/famedly/fluffychat/-/merge_requests/1111). There is https://gitea.blesmrt.net/mikaela/gist/src/branch/master/matrix/FluffyChat/quick-account-switching.md as for documentation on how it may be used, but I would not be aware of any outside that. As OP said, the feature is, perhaps, rather obscure.

**Comment by kfiven** [comment]
duplicate of #66

**Comment by viasux** [comment]
> duplicate of #66

I don't think it's quite a duplicate, I believe OP was requesting similar functionality to how pluralkit works on discord.

**Comment by theotheroracle** [comment]
i was yea

**Comment by kfiven** [comment]
Well then, I need to how's that different from normal multi account functionality. I'll reopen till then.

**Comment by theotheroracle** [comment]
if you start a message with a prefix, like "b." then it would send a message as the account linked to that prefix

**Comment by viasux** [comment]
There's a nice gif on their homepage https://pluralkit.me/

**Comment by kfiven** [comment]
Looks like this can be done with [MSC4144](https://github.com/matrix-org/matrix-spec-proposals/pull/4144).
